src/site/content/en/blog/sms-receiver-announce/index.md

```diff
@@ -0,0 +1,401 @@
+
+These days, most people in the world own a mobile device with a phone number,
+and developers are commonly using a phone number as an identifier for users of
+their services.
+
+There are a variety of ways to verify phone numbers, but a randomly generated
+one-time password (OTP) sent by SMS to the number is the most common. Sending
+this code back to the developer's server demonstrates control of the phone
+number.
+
+This idea is already deployed in many online scenarios to achieve:
+
+* **Phone number as a user's primary identifier:** When signing up for a new
+  service, some websites ask for a phone number instead of an email address and
+  use it as their primary identifier. It's important to verify phone numbers in
+  this use case.
+* **Two step verification:** When signing in, a website asks for a one-time code
+  sent via SMS on top of a password for extra security.
+* **Payment confirmation:** When a user is making a payment, asking a one-time
+  code sent via SMS can help authenticate the person's intent.
+
+The current process creates friction for the user. Finding an OTP within an SMS
+message, then copying and pasting it to the form is cumbersome. Easing this has
+been a long standing request for the web from many of the largest global
+developers. Android has [an API that does exactly
+this](https://developers.google.com/identity/sms-retriever/). So does
+[iOS](https://developer.apple.com/documentation/security/password_autofill/about_the_password_autofill_workflow)
+and
+[Safari](https://developer.apple.com/documentation/security/password_autofill/enabling_password_autofill_on_an_html_input_element).
+
+With the SMS Receiver API, you can programmatically obtain an OTP from an SMS
+message and verify a phone number for the user more easily.
+
+{% Aside 'warning' %}
+keep in mind that SMS is a weak form of authentication: attackers can
+spoof SMS and can hijack a person's phone number. Carriers can also recycle
+phone numbers to new users after an account was closed. While SMS OTP is useful
+to verify a phone number when a user sets up an account with your service, we
+recommend using stronger forms of authentication, such as multiple factors and
+[WebAuthn](https://www.w3.org/TR/webauthn/) to establish new sessions for these
+users.
+{% endAside %}
+
+## Current status
+
+<table>
+<tr>
+<td markdown="block">
+Step
+</td>
+<td markdown="block">
+Status
+</td>
+</tr>
+<tr>
+<td markdown="block">
+1. Create explainer
+</td>
+<td markdown="block">
+<a href="https://github.com/samuelgoto/sms-receiver/blob/master/README.md" >Complete</a>
+</td>
+</tr>
+<tr>
+<td markdown="block">
+2. Create initial draft of specification
+</td>
+<td markdown="block">
+In Progress
+</td>
+</tr>
+<tr>
+<td markdown="block">
+3. Gather feedback & iterate on design
+</td>
+<td markdown="block">
+In Progress
+</td>
+</tr>
+<tr>
+<td markdown="block">
+<strong>4. Origin trial</strong>
+</td>
+<td markdown="block">
+<strong>Starts in Chrome 78</strong><br/>
```

Comment:
Can you add a link to the OT (https://developers.chrome.com/origintrials/#/view_trial/607985949695016961) please